message-index/messages/GHC-11861/example1/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Empty character literals
+summary: Empty character is not a valid literal 
+introduced: 9.6.1
+severity: error
+---
+
+Empty character was given in a single quote. This is not a valid literal. If you want to pass a whitechar use explicit space character `' '` or other from the list: newline, vertab, space, tab, uniWhite.
```

Comment:
```suggestion
An empty pair of single quotes is not a valid literal. To indicate a whitespace character, either use an explicit space character `' '` or an escape code such as `'\n'` for a newline.
```
I think that this phrasing is easier to follow, because it sounded to me like a newline character literal could be as follows:
```
foo = '
'
```